.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
If we are removing blank issues, shall we add a template for TODO's? I've often seen blank issues being helpful for those, since they don't always perfectly fit under "feature requests" or "bug reports"

Reply:
Hi Vir, no, I don't think we need a separate template for TODOs. What's an example of a TODO that wouldn't be either a feature (an extension of functionality) or a bug (something not working optimally that needs fixing)?

Reply:
For example, something like moving a feature flag from test stage to prod stage -- it's not really a feature request or a bug